Access error: Auth name cannot be updated for SSO users (#4145)
cloudflare/cloudflare-docs: content/cloudflare-one/faq/teams-troubleshooting.md

If `cloudflared` returns error `error="remote error: tls: handshake failure"`, check to make sure the hostname in question is covered by a SSL certificate. If using a multi-level subdomain, an advanced certificate may be required as the Universal SSL will not cover more than one level of subdomain. This may surface in the browser as `ERR_SSL_VERSION_OR_CIPHER_MISMATCH`.

> ## Access API error auth_domain_cannot_be_updated_dash_sso.

Reviewer: ```suggestion
## I see `Access api error auth_domain_cannot_be_updated_dash_sso`.
```

> You may see this error while trying to change your [ZT auth domain](https://developers.cloudflare.com/cloudflare-one/faq/teams-getting-started-faq/#whats-a-team-domainteam-name) aka "Team domain" and SSO is enabled for your account.

Reviewer: ```suggestion
You may see this error while trying to change your [team domain](/cloudflare-one/faq/teams-getting-started-faq/#whats-a-team-domainteam-name). This error occurs because your account is using the [Cloudflare Dashboard SSO application](/cloudflare-one/applications/configure-apps/dash-sso-apps/).
```

> Currently customers that have [SSO](https://developers.cloudflare.com/cloudflare-one/applications/configure-apps/dash-sso-apps/) enabled, cannot change their ZT auth domain, contact support for more details.

Reviewer: ```suggestion
Currently, customers who have the SSO feature enabled cannot change their team domain. Contact support for more details.
```
